code/game/machinery/doors/door.dm

```diff
@@ -97,6 +98,10 @@
 			return TRUE
 	return FALSE
 
+/// Triggered when attempting to remove an airlock seal
+/obj/machinery/door/proc/try_remove_seal(mob/user)
+	return
```

Comment:
Could use improved documentation as per our [documentation guidelines](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)

Please add information on the user argument for try_remove_seal. Doesn't have to be long, as long as it's documented as per the guidelines.

Reply:
I think that's resolved